### PR TITLE
specify number of 100s/50s in !acc

### PR DESCRIPTION
**Pull request by omkelderman**
As discussed in #136.

Implementation *probably* works, but I can't test it since the testbackend-stuff doesnt properly implement `PercentageEstimates`.

Line 102 in `TestBackend.java` for example, did a random thing to make it happy, but its probably not what it should be :stuck_out_tongue:

### CODE REVIEW

**Comment by omkelderman** [comment]
I merged in master with the new estimates code n stuff, so now there is no awkward testbackend stuff going on anymore 😄 

Still, this *probably* works, but can't say for sure till tested with real data, which the testbackend isnt good enough for. So @Tillerino you can merge it once you think it works. I think it does, but im not 100% sure.

**Comment by omkelderman** [comment]
got triggered by it, so I committed another thing xD

File: tillerinobot/src/main/java/tillerino/tillerinobot/diff/PercentageEstimates.java [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
 	public double getPP(double acc, int combo, int misses);
 
+	public double getPPForHitPoints(int x100, int x50, int combo, int misses);
```
Comment (Tillerino) [inline tillerinobot/src/main/java/tillerino/tillerinobot/diff/PercentageEstimates.java]: "HitPoints"?

Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/diff/PercentageEstimates.java]: idk xD its what I call the 50's and 100's I guess, no idea, I cant name things

Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/diff/PercentageEstimates.java]: could just not bother and call it `getPP` cuz params are different anyway

Comment (Tillerino) [inline tillerinobot/src/main/java/tillerino/tillerinobot/diff/PercentageEstimates.java]: getPpForAccuracy and getPpForHits?

Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/diff/PercentageEstimates.java]: sure, that works. Guess i'll have to rename my branch as well xD

Comment (omkelderman) [inline tillerinobot/src/main/java/tillerino/tillerinobot/diff/PercentageEstimates.java]: altho tbh, I dont think my idea of just calling it `getPP` and let the parameters speak for itself isnt such a bad idea

**Comment by Tillerino** [comment]
Is this ready?

**Comment by omkelderman** [comment]
not anymore LUL
It was, even including the "triggered" commit, that was just moving a random check done somewhere outside an ifelse thing so it only does it once.
But its broken now, cuz it has conflicts cuz you merged my other PR haha
will fix in a sec

**Comment by omkelderman** [comment]
if travis is happy, and more importantly, if *you* are happy, I'd say this is ready.

Keep in mind, I dont know 100% for sure this works, because testbackend isnt real map-data. But I'm pretty confident it will *probably* work 😉

**Pull request merged**